tests/sdl/whitelists.py

```diff
@@ -0,0 +1,223 @@
+#
+# Copyright (c) 2023 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from enum import Enum, auto
+
+
+class OvmsImageType(Enum):
+    UBUNTU_GENERIC = auto()
+    UBUNTU_22_GENERIC = auto()
+    UBUNTU_22_GPU = auto()
+    UBUNTU_CUDA = auto()
+    UBUNTU_NGINX = auto()
+    REDHAT_GENERIC = auto()
+    REDHAT_GPU = auto()
+    REDHAT_CUDA     = auto()
+
+
+# DYNAMIC LIBRARIES
+dynamic_libraries_common = {
+    'libgcc_s.so', 'liblzma.so', 'libstdc++.so', 'libuuid.so', 'libxml2.so'
+}
+ 
+dynamic_libraries_ubuntu = {'libicuuc.so', 'libicudata.so',}
+
+dynamic_libraries_ubuntu_python = {'libexpat.so',}
+
+dynamic_libraries_ubuntu_20_04 = dynamic_libraries_ubuntu | {'librt.so', 'libtbb.so',}
+
+dynamic_libraries_ubuntu_20_04_python = dynamic_libraries_ubuntu_python | {'libpython3.8.so', 'libutil.so',}
+
+dynamic_libraries_ubuntu_22_04 = dynamic_libraries_ubuntu | {'libm.so', 'libdl.so', 'libpthread.so',}
+
+dynamic_libraries_ubuntu_22_04_pythnon = dynamic_libraries_ubuntu_python | {'libpython3.10.so',}
+
+dynamic_libraries_ubuntu_22_04_no_python = {'libpugixml.so',}
+
+dynamic_libraries_redhat = {'libtbb.so',}
+
+dynamic_libraries_redhat_python = {'libpython3.9.so', 'libutil.so',}
```

Comment:
Can we use single map of sets to store those libs?
Usage of several variables isn't flexible

Reply:
fixed